libs/openFrameworks/utils/ofFileUtils.cpp

```diff
@@ -1775,7 +1775,8 @@ std::string ofFilePath::getEnclosingDirectory(const of::filesystem::path & _file
 	if(bRelativeToData){
 		fp = ofToDataPath(fp);
 	}
-	return addTrailingSlash(fp.parent_path());
+//	return addTrailingSlash(fp.parent_path());
+	return fp.parent_path().string();
```

Comment:
This change seems problematic - could break a lot of stuff

Reply:
Yes, for sure. I've reverted back